octavia_f5/restclient/as3objects/pool.py

```diff
@@ -73,6 +78,8 @@ def get_pool(pool):
                     member_hm.set_target_port(member.monitor_port)
                 name = m_member.get_name(member.id)
                 entities.append((name, member_hm))
+                if not utils.pending_delete(member):
+                    service_args['monitors'].append(Pointer(use=name))
```

Comment:
Please add comment hinting to this being part of the workaround, similar to line 72.

Reply:
this is not part of the workaround, but of the custom member address/port. just a fix